Vagrantfile

```diff
@@ -56,7 +56,7 @@ Vagrant.configure(2) do |config|
     config.vm.provision "shell", inline: "apt-get update"
 
     # Install build dependencies
-    config.vm.provision "shell", path: "cookbook/system-config/setup-ccache-and-icecc.sh"
+#    config.vm.provision "shell", path: "cookbook/system-config/setup-ccache-and-icecc.sh"
```

Comment:
I think we might want to enable it fairly soon again if we can get icecc stable. This is a bit  unrelated to the story tbh, but icecc has become unreliable after the changes to the network for some reason.